checkov/terraform/checks/resource/azure/AppConfigReplicaEnabled.py

```diff
@@ -0,0 +1,23 @@
+from checkov.common.models.consts import ANY_VALUE
+
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from typing import Any
+
+
+class AppConfigReplicaEnabled(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure App configuration has at least one replica configured."
+        id = "CKV_AZURE_242"
```

Comment:
This check id already taken for:
"Ensure isolated compute is enabled for Synapse Spark pools"